README.md

```diff
@@ -1,2 +1,34 @@
 Ethereum On-Boarding Testsuite
-==============================
\ No newline at end of file
+==============================
+
+## Implement a Single Node Ethereum Test Network
+
+1. Create an account on https://kurtosistech.com if you don't have one yet.
+2. Verify that the Docker daemon is running on your local machine.
+3. Clone this repository by running `git clone https://github.com/kurtosis-tech/kurtosis-onboarding-experience.git`
```

Comment:
Let's add `--branch master` to the `clone` command so the user gets the master branch